docs/userGuide/plugins/googleAnalytics.mbdf

```diff
@@ -6,7 +6,7 @@ To enable it, add `googleAnalytics` to your site's plugins, and add the `trackin
 
 Name | Type | Default | Description
 ---- | ---- | ------- | ------
-trackingID | `String` || Tracking ID provided by Google. Follow this [guide](https://support.google.com/analytics/answer/1008080) to get your Tracking ID.
+trackingID | `String` || Tracking ID is provided by Google. Follow this [guide](https://support.google.com/analytics/answer/1008080) to get your Tracking ID.
```

Comment:
👀

Reply:
Hmm I was thinking the sentence might have missed out a "is" but now that I think about it again, it might have been intended that way due to the context. Can I confirm if this is the case? If so, I will remove it!

Reply:
@wxwxwxwx9 I suppose you may submit a separate PR for such changes in the future - as this change is not that relevant to this PR.

Reply:
Thanks for the tip @le0tan! Will keep it in mind for the future.